### ISSUE TITLE
Create visual products from FMRC UGRID datasets

### ISSUE THREAD

**Issue by lesserwhirls** [ioos/thredds-ugrid#10]
Create basic maps, charts, etc. of various fields within UGRID datasets as exposed by the TDS via FMRC.

**Comment by dpsnowden** [comment]
Does this include extracting a time series from a grid at a node and at a lat/lon?

**Comment by lesserwhirls** [comment]
Yep, this would include visuals supported by the use of subsetting in geophysical space, including single lat/lon time series. I've updated the checklist on ioos/thredds-ugrid#6 to add this specific case. If you can think of more, leave a comment over there and I'll make sure it gets added to the list!
